### PR TITLE
feat: new icon in preferences, svg icons demo, clean out css

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/demo/template/icons.htm [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <mute-icon></mute-icon>
+    <people-icon></people-icon>
+    <pickup-icon></pickup-icon>
+    <plus-icon></plus-icon>
+    <profile-icon></profile-icon>
+    <screenshare-icon></screenshare-icon>
+    <settings-icon></settings-icon>
+    <undo-icon></undo-icon>
+    <verified-icon class="override-color"></verified-icon>
+</div>
```
Comment (gregor) [inline app/demo/template/icons.htm]: Please add empty line at eof

Comment (gregor) [reply app/demo/template/icons.htm]: Do we have any header in the html files?

File: app/style/fonts/zeta-neue.css [hunk 2]
```diff
@@ -267,21 +213,12 @@
...
 }
-.icon-checkbox:before {
-  content: "\e600";
-}
-.icon-checkbox-checked:before {
-  content: "\e601";
-}
 .icon-contacts:before {
   content: "\e602";
 }
```
Comment (gregor) [inline app/style/fonts/zeta-neue.css:224]: We shouldn't really manually edit this file. Design creates it as the output when we generated a new icon font. How much work left to remove it completely?

File: app/demo/template/icons.htm (hunk 1 above)
Comment (AndyLnd) [reply app/demo/template/icons.htm]: These files don't have any headers as they are just fragments.

File: app/style/fonts/zeta-neue.css (hunk 2 above)
Comment (AndyLnd) [reply app/style/fonts/zeta-neue.css:224]: This file won't see any automatic updates anymore. All new icons now come as svg. Removing the obsolete ones is a good way of keeping track of missing svg implementations and to-be-replaced icons.

Comment (gregor) [reply app/style/fonts/zeta-neue.css:224]: I don't fully agree but let's go with it. I see removing the icon font as one task that will be done at the end if one chance of breaking stuff. Now we possibly break things everytime we add a new SVG based icon and forget about an instance of the old icon font based one.

**Pull request merged**